themes/10up-block-theme/src/Assets.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Assets module.
+ *
+ * @package TenupBlockTheme
+ */
+
+namespace TenupBlockTheme;
+
+use TenupFramework\Module;
+use TenupFramework\ModuleInterface;
+use TenupBlockTheme\Traits\GetAssetInfo;
+
+/**
+ * Assets module.
+ *
+ * @package TenupBlockTheme
+ */
+class Assets implements ModuleInterface {
+
+	use Module;
+	use GetAssetInfo;
+
+	/**
+	 * Can this module be registered?
+	 *
+	 * @return bool
+	 */
+	public function can_register() {
+		return true;
+	}
```

Comment:
Stupid question: 

Would it not be better to have this return true by default in the `ModuleInterface` base class? In 90% of cases that is what we want. And when we don't we can override it. But it seems redundant to always have to add this to every single file 🤔 (very loosely held opinion)

Reply:
I agree, though that's a change we'll need to make in the Framework and then update here 🙂